mne_bids/mne_bids.py

```diff
@@ -670,5 +670,15 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         copyfile_eeglab(raw_fname, raw_file_bids)
     else:
         sh.copyfile(raw_fname, raw_file_bids)
+    # KIT data requires the marker file to be copied over too
+    if manufacturer == 'KIT/Yokogawa' and hpi is not None:
```

Comment:
> we don't support list of hpis in general so we can remove the manufacturer logic above

so you are saying we only need to check `if hpi is not None`? (and probably refactor into line below...)

Reply:
also if we specify that the `hpi` can only be a string then we don't need to raise an error about it being a list

Reply:
good point. I would say we should have str compatible only and depending on the decision on bids-standard, we could expand the type support (str, list) or just leave it at str

Reply:
so why not just leave it with support for lists (as in leave code as-is (other than remove manufacturer logic)) for when a decision is made, otherwise we will need to re-allow list usage.

Reply:
the averaging and list behavior is part of mne-python. I am just suggesting that we just change the docstring to restrict list usage that way. it would be any code changing.

Reply:
ok, I'll remove the manufacture check and remove the mention of lists in the docstring, but leave the check for a list in the code with a note about the issue in bids-standard?

Reply:
sure, that works for me